virtcontainers/qemu.go

```diff
@@ -833,7 +833,7 @@ func (q *qemu) hotplugVFIODevice(device *config.VFIODev, op operation) error {
 	return nil
 }
 
-func (q *qemu) hotplugMacvtap(drive *VirtualEndpoint) error {
+func (q *qemu) hotplugMacvtap(drive *VethEndpoint) error {
```

Comment:
This function is for macvtap

Reply:
@caoruidong Yes, but the underlying interface is the Veth interface. Macvtap is used as the bridging mechanism. Answering your previous question, the difference is when you get a macvtap interface from the network plugin that you can directly pass to the VM vs an interface such as a veth that you get from the plugin wherein you use a macvtap or a bridge+tap for bridging the interface to the VM.

Reply:
Oh, so the new MacvtapEndpoint is without veth pair? Got it.